packages/use-intl/src/core/IntlError.tsx

```diff
@@ -5,7 +5,8 @@ export enum IntlErrorCode {
   INSUFFICIENT_PATH = 'INSUFFICIENT_PATH',
   INVALID_MESSAGE = 'INVALID_MESSAGE',
   INVALID_KEY = 'INVALID_KEY',
-  FORMATTING_ERROR = 'FORMATTING_ERROR'
+  INVALID_FORMAT = 'INVALID_FORMAT',
```

Comment:
I thought so at first but then I thought FORMATTING_ERROR was a formatting error but not an error because of the format while parsing. but if it still makes sense to you I'll revert it